### PR TITLE
Feature/crpl 2834 add binance pancake deploy and helpers

### PR DESCRIPTION
**Pull request by gyhevizi**

### CODE REVIEW

File: blockchain/ethereum/pancakeswap/wbnb deployer/0953af1d-ae69-47fe-aa66-085699649f88.json [hunk 1]
```diff
@@ -0,0 +1,111 @@
+{
+  "name": "blockchain/ethereum/pancakeswap/wbnb deployer",
```
Comment (csabee1979) [inline blockchain/ethereum/pancakeswap/wbnb deployer/0953af1d-ae69-47fe-aa66-085699649f88.json:2]: should have a more readable name. Eg Wraped BNB token deployer. The node name needs to start upper case.

File: blockchain/ethereum/pancakeswap/wbnb deployer/0953af1d-ae69-47fe-aa66-085699649f88.json [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+          }
+        }
+      }
+    }
+  },
+  "iconId": "abac34e3-c832-4c7b-b045-62f82ebc3ddb",
+  "original": "6a8ac46e-756f-4028-bf83-e5fd7fdd8788",
+  "keywords": {
+    "erc20": null,
+    "token": null
```
Comment (csabee1979) [inline blockchain/ethereum/pancakeswap/wbnb deployer/0953af1d-ae69-47fe-aa66-085699649f88.json:106]: Additional keywords: e.g binance, bnb, bsc

**Review by csabee1979 — CHANGES_REQUESTED** [review]
The node name needs to start upper case

File: blockchain/ethereum/pancakeswap/pancakeswap router deployer/a9de8194-778e-4284-b46d-dd1686c574b3.json [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+  "iconId": "abac34e3-c832-4c7b-b045-62f82ebc3ddb",
+  "original": "e9af93a1-4171-4683-bbfa-f6379bcf011e",
+  "keywords": {
+    "token": null,
+    "erc20": null,
+    "dex": null,
+    "Pancake": null,
+    "Uniswap": null
+  },
+  "attributes": {}
```
Comment (csabee1979) [inline blockchain/ethereum/pancakeswap/pancakeswap router deployer/a9de8194-778e-4284-b46d-dd1686c574b3.json:251]: Should be locked.

File: blockchain/ethereum/pancakeswap/pancakeswap router deployer/a9de8194-778e-4284-b46d-dd1686c574b3.json [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+            }
+          }
+        }
+      }
+    }
+  },
+  "iconId": "abac34e3-c832-4c7b-b045-62f82ebc3ddb",
+  "original": "e9af93a1-4171-4683-bbfa-f6379bcf011e",
+  "keywords": {
+    "token": null,
```
Comment (csabee1979) [inline blockchain/ethereum/pancakeswap/pancakeswap router deployer/a9de8194-778e-4284-b46d-dd1686c574b3.json:245]: Additional keywords: e.g binance, bnb, bsc

File: blockchain/ethereum/demo/demo pool creator/9b8748db-7f0b-4304-b79a-d65a473f255f.json [hunk 5]
```diff
@@ -0,0 +1,299 @@
+{
+  "implementations": {
+    "cranq": {
+      "nodes": {
+        "144e4964-bf4e-48f7-8bb1-11551108e472": {
```
Comment (csabee1979) [inline blockchain/ethereum/demo/demo pool creator/9b8748db-7f0b-4304-b79a-d65a473f255f.json:5]: Failure while loading prototype. Child prototype definitions may be missing, or the prototype might be corrupted. Please try updating your repositories.

[Error] prototype `f2c2b6c4-ddb5-451d-945e-f37457ccf2e6` of output node `100 thousand x 10^18` (64edc4a0-9179-4d53-b761-9621b8a239ec) cannot be found

File: blockchain/ethereum/demo/demo pool creator/9b8748db-7f0b-4304-b79a-d65a473f255f.json [hunk 6]
```diff
@@ -0,0 +1,299 @@
...
+      }
+    },
+    "outputs": {
+      "6760cac1-88cd-4079-9e17-6093026e23b4": {
+        "type": "any",
+        "name": "out"
+      }
+    }
+  },
+  "name": "blockchain/ethereum/demo/demo pool creator",
```
Comment (csabee1979) [inline blockchain/ethereum/demo/demo pool creator/9b8748db-7f0b-4304-b79a-d65a473f255f.json:293]: I am not sure the demo name space is the proper name.

[the remark this answers is not in the document]
Comment (gyhevizi) [reply blockchain/ethereum/demo/demo pool creator/9b8748db-7f0b-4304-b79a-d65a473f255f.json]: Discussed yesterday, have no better candidate so far.

File: blockchain/ethereum/demo/demo pool creator/9b8748db-7f0b-4304-b79a-d65a473f255f.json (hunk 5 above)
Comment (gyhevizi) [reply blockchain/ethereum/demo/demo pool creator/9b8748db-7f0b-4304-b79a-d65a473f255f.json:5]: added yesterday

**Review by csabee1979 — APPROVED** [review]

**Pull request merged**